gcloud/monitoring/client.py

```diff
@@ -195,6 +203,121 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
```

Comment:
While I prefer the way I did it, if there's other cases of this in the repo, then it's not a big deal to me. 

The case you gave of Datastore Entity is just a pass through from the client module to the constructor of the other module. For Metric and Resource this is no problem. For TimeSeries, we really do want a factory function, and I'm not sure if we want to be changing TimeSeries itself, and it being a namedtuple also slightly complicates things. Do I write a factory function in `timeseries.py` then import that as `TimeSeries` into `client`? Should I just be more explicit about the fact that it is a factory and not a constructor? A few ways to do it, would really like to hear from @rimey  on the issue before I write more code that may need to be rewritten.